ironfish/src/mining/manager.ts

```diff
@@ -194,7 +186,6 @@ export class MiningManager {
 
     this.blocksMined++
     this.onNewBlock.emit(block)
-    this.telemetry.submitBlockMined(block)
```

Comment:
I did this in this PR because we need to be able to access `peerNetwork.localPeer.publicIdentity` in telemetry. So I had to instantiate telemetry after the peerNetwork which meant after the miningManager (since peerNetwork is dependent on mining manager). I think this is probably better as well so we don't have miningManager dependent on telemetry

Reply:
That sounds reasonable.